etc/config/csharp.amazon.properties

```diff
@@ -23,6 +23,6 @@ compiler.dotnet700csharp.langVersion=latest
 compiler.dotnettrunkcsharp.exe=/opt/compiler-explorer/dotnet-trunk/.dotnet/dotnet
 compiler.dotnettrunkcsharp.name=.NET (main)
 compiler.dotnettrunkcsharp.clrDir=/opt/compiler-explorer/dotnet-trunk
-compiler.dotnettrunkcsharp.targetFramework=net8.0
+compiler.dotnettrunkcsharp.targetFramework=net7.0
```

Comment:
@mattgodbolt, is it possible to automate this property with https://github.com/dotnet/runtime/blob/main/global.json#L3. global.json's format hasn't changed since .net core v1, so reading first three characters of `sdk.version` (`7` `.` `0`) and prepending `net` will save us from updating `main` manually.